.circleci/config.yml

```diff
@@ -247,8 +276,8 @@ jobs:
 
       # union the bundle targets until //... doesnt explode analyzing jvm/android targets
       - run: |
-          BUNDLE_TARGETS=$(bazel query "kind(js_test, //plugins/...) union kind(js_test, //react/...) union kind(js_test, //core/...)" --output label 2>/dev/null | tr '\n' ' ')
-          bazel coverage --combined_report=lcov --config=ci -- $BUNDLE_TARGETS -//plugins/reference-assets/mocks:all_flows_test_binary 
+          BUNDLE_TARGETS=$(bazel query "kind(js_test, //plugins/...) union kind(js_test, //react/...) union kind(js_test, //core/...) union kind(test, //jvm/...) union kind(test, //android/...) - //android/demo:android_instrumentation_test" --output label 2>/dev/null | tr '\n' ' ')
```

Comment:
This doesn't account for jvm and android tests under `//plugins`. From the above:
> union the bundle targets until //... doesnt explode analyzing jvm/android targets
We could probably get away with something closer to `//...` and minus all the ios and instrumented android tests. I just don't know how complicated the union is going to be to match all the tests we want